crates/oxc_cli/src/main.rs

```diff
@@ -21,7 +21,8 @@ fn main() -> CliRunResult {
                 return CliRunResult::PathNotFound { path: path.clone() };
             }
 
-            Cli::lint(path)
+            let quiet = matches.get_one::<String>("quiet").is_some();
+            Cli::lint(path, CliOptions { quiet })
```

Comment:
We'll end up with 
```rust
let cli_options = CliOptions::From(clap matches);
let cli = Cli::new(cli_options);
cli.lint();
```

This should scale once we get even more top-level args and subargs.